src/metrics/http_metrics.rs

```diff
@@ -0,0 +1,72 @@
+use std::fmt::Write;
+
+use prometheus_client::encoding::{EncodeLabelSet, EncodeLabelValue, LabelValueEncoder};
+use prometheus_client::metrics::counter::Counter;
+use prometheus_client::metrics::family::Family;
+use prometheus_client::metrics::histogram::{linear_buckets, Histogram};
+use prometheus_client::registry::Registry;
```

Comment:
This file should also be formatted with `cargo fmt`.

Reply:
Formatted with `cargo fmt`.